app/jobs/update_schools_data_from_source_job.rb

```diff
@@ -82,5 +88,7 @@ def csv_url
 
   def valid_website(url)
     Addressable::URI.heuristic_parse(url).to_s
+  rescue Addressable::URI::InvalidURIError
```

Comment:
Another fix we could cherry-pick onto a new branch to keep along with it's test and get merged in asap.